MAINTAINERS.md

```diff
@@ -1,2 +1,3 @@
 * Ben Kochie <superq@gmail.com> @SuperQ
+* Conrad Hoffmann <> @bitfehler
```

Comment:
```suggestion
* Conrad Hoffmann <ch@bitfehler.net> @bitfehler
```

Either way, :shipit:

Reply:
Just didn't know what email to use for this. :)